test/ServiceManagerTest.php

```diff
@@ -364,4 +365,51 @@ public function testResolvedAliasNoMatchingAbstractFactoryReturnsFalse()
 
         self::assertFalse($serviceManager->has('Alias'));
     }
+
+    /**
+     * Hotfix #3
+     * @see https://github.com/laminas/laminas-servicemanager/issues/3
+     */
+    public function testConfigureMultipleTimesAvoidsDuplicates()
+    {
+        $delegatorFactory = function (
+            ContainerInterface $container,
+            $name,
```

Comment:
Should probably have `string` here

Reply:
I've copied this code from the original repo and cast my eye over it for sanity but I absoltely could add type declarations if we think that's appropriate. My thought was to keep it in line with the existing code which doesn't have type declarations but it's a good chance to start moving in that direction if we think it's a good time to start.

How would we feel about some methods having types and others not?

Reply:
If we're bumping the PHP version to v3, I'd argue _add them everywhere_ in the test suite.

Reply:
Yeh, but then not in this patch.

We can run psalter manually in a different patch.

Discussed this with @GeeH and it should be fine as-is for now